src/Config/RuntimeConfigLoader.cs

```diff
@@ -101,4 +110,43 @@ public static JsonSerializerOptions GetSerializationOptions()
         options.Converters.Add(new StringJsonConverterFactory());
         return options;
     }
+
+    /// <summary>
+    /// It adds or replaces a property in the connection string with `Application Name` property.
+    /// If the connection string already contains the property, it uses a regular expression to update the existing value
+    /// by adding the DataApiBuilder Application Name (dab_oss/dab_hosted). If not, it appends the property `Application Name` to the connection string.
+    /// This method only adds the `Application Name` property for MSSQL, as other DB's have different ways to specify the Application Name. 
+    /// </summary>
+    /// <param name="databaseType">Type of Database</param>
+    /// <param name="connectionString">Connection string for connecting to database.</param>
+    /// <returns></returns>Updated connection string with `Application Name` property<summary>
+    public static string GetConnectionStringWithApplicationName(DatabaseType databaseType, string connectionString)
+    {
+        if (databaseType is not DatabaseType.MSSQL || string.IsNullOrWhiteSpace(connectionString))
```

Comment:
if this function is only for MSSQL, shouldn't it be the callers' responsibility to call it only for that case? why would we add the argument DatabaseType and an additional call overhead for other types?